recipes/azure-purview-workflow/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "azure-purview-workflow" %}
+{% set version = "1.0.0b1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/azure-purview-workflow-{{ version }}.zip
+  sha256: e55559a414e80b4e196ff3295b2c7ad2a2bbbb6502156631512297b7f694beb4
+
+build:
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - isodate <1.0.0,>=0.6.1
+    - azure-core <2.0.0,>=1.24.0
+    - typing-extensions >=4.3.0  # [py<38]
```

Comment:
Would be preferable to unconditionally depend on `typing-extensions` and make this noarch.